src/Workspaces/CoreTestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -384,6 +384,9 @@ public override int WriteTimeout
 
                 public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken) => _stream.CopyToAsync(destination, bufferSize, cancellationToken);
 
+#if NET5_0
+                [Obsolete]
```

Comment:
I saw the tfm warnings, but what issue is this fixing?

Reply:
The warning not being reported.

Reply:
Seems like the newer compiler is reporting more warnings than the previous version.

Reply:
Odd, now I see 
```
error CS0809: Obsolete member 'InProcRemoteHostClient.InProcRemoteServices.WrappedStream.InitializeLifetimeService()' overrides non-obsolete member 'MarshalByRefObject.InitializeLifetimeService()
```

Reply:
Turn out we can remove this override.